src/components/DesingSpectrumChart/DesingSpectrumChart.jsx

```diff
@@ -0,0 +1,16 @@
+import Highcharts from "highcharts";
+import HighchartsReact from "highcharts-react-official";
+import chartOptions from "./DSChartOptions";
+import { useState } from "react";
+
+const DesingSpectrumChart = () => {
+  const [ options, setOptions ] =useState(chartOptions)
+  return (
+    <>
+      <h1>Desing Spectrum</h1>
+      <HighchartsReact highcharts={Highcharts} options={chartOptions}/>
+    </>
```

Comment:
Here, property options must be pairing with option of local state or i'm wrong?

Reply:
Yes, I'll check if it can be mutated to add the arrays (chart lines.)